Src/Newtonsoft.Json-for-Unity/README.md

```diff
@@ -76,8 +92,10 @@ Change the version field. You have to know the new version beforehand.
 This package is licensed under The MIT License (MIT)
 
 Copyright (c) 2019 Kalle Jillheden (jilleJr)  
-<https://github.com/jilleJr/Newtonsoft.Json>
+<https://github.com/jilleJr/Newtonsoft.Json-for-Unity>
 
 See full copyrights in [LICENSE.md][license.md] inside repository
 
 [license.md]: https://github.com/jilleJr/Newtonsoft.Json-for-Unity/blob/master/LICENSE.md
+[changelog.md]: https://github.com/jilleJr/Newtonsoft.Json-for-Unity/blob/master/LICENSE.md
+[version-explanation]: https://github.com/jilleJr/Newtonsoft.Json-for-Unity/raw/ce23d98230673744d73656b4c4f6bc1f9989c37a/Doc/version-explanation.png
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-unused-definitions] Found unused definition](https://app.codacy.com/app/jilleJr/Newtonsoft.Json-for-Unity/pullRequest?prid=4554595)